src/components/lightDarkTheme/LightDarkTheme.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect, useCallback } from 'react';
+
+export const LightDarkThemeToggle = () => {
+  const defaultTheme = window.matchMedia('(prefers-color-scheme: dark)')?.matches ? 'dark' : 'light';
```

Comment:
Good practice is move such string as "dark" and "light" in constants. The same as for options for dropdowns, labels and such stuff.  
Let's do it later.

Reply:
have some issues with path. Kindly check my last commit "some architecture changes"